ndohyep/static/css/feature.css

```diff
@@ -60,7 +64,7 @@ blockquote.yellow {color:#fec642;}
 .grey a, .grey h2, .grey .author {color:#aaaaaa;}
 .orange a, .orange h2, .orange .author {color:#f26d40;}
 .yellow a, .yellow h2, .yellow .author {color:#fdc741;}
-.blue a, .blue h2, .blue .author {color:#60a2ec;}
+.blue a, .blue h2, .blue .author {color:white;}
```

Comment:
@bruskiza @Pringels I think this is going to be a problem. Because `<div class='blue'><h2>I am blue</h2></div>` is actually going to be white..